makerspace/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path, register_converter
+
+from makerspace import views
+
+urlpatterns = [
```

Comment:
Raw strings are not needed as they only escape backslashes, which should in general never be used in `urls`. It is true that previously, Django used regexes for their urls, but unless you use `re_path` this is no longer the case. So using raw strings gives no additional gain and are in general not used for the `path` function.